ED/qa.html

```diff
@@ -0,0 +1,1004 @@
+                  <p property="skos:definition">The <span about="" rel="spec:specificationCategory" resource="#specification-category">Solid Notifications Protocol</span> identifies the following <cite><a href="https://www.w3.org/TR/spec-variability/#spec-cat" rel="dcterms:subject" resource="spec:SpecificationCategory">Specification Category</a></cite> to distinguish the types of conformance: <span rel="skos:hasTopConcept" resource="spec:API">API</span>, <span rel="skos:hasTopConcept" resource="spec:NotationSyntax">notation/syntax</span>, <span rel="skos:hasTopConcept" resource="spec:SetOfEvents">set of events</span>, <span rel="skos:hasTopConcept" resource="spec:ProcessorBehaviour">processor behaviour</span>, <span rel="skos:hasTopConcept" resource="spec:Protocol">protocol</span>.</p>
+                </div>
+              </section>
+
+              <section id="classes-of-products" inlist="" rel="schema:hasPart" resource="#classes-of-products" typeof="skos:ConceptScheme">
+                <h3 property="schema:name skos:prefLabel">Classes of Products</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p property="skos:definition">The <span about="" rel="spec:classesOfProducts" resource="#classes-of-products">Solid QA</span> identifies the following <cite><a href="https://www.w3.org/TR/qaframe-spec/#cop-def" rel="dcterms:subject" resource="spec:ClassesOfProducts">Classes of Products</a></cite> for conforming implementations. These products are referenced throughout this specification.</p>
+
+                  <span rel="skos:hasTopConcept"><span resource="#TestReport"></span></span>
+
+                  <dl>
+                    <dt about="#TestReport" property="skos:prefLabel" rel="skos:relatedMatch" resource="spec:RespondingAgent" typeof="skos:Concept"><dfn id="TestReport">Test Report</dfn></dt>
+                    <dd about="#TestReport" property="skos:definition">A <a href="#test-report" rel="rdfs:seeAlso">Test Report</a> is a resource that describes the level of conformance of a project tests against <a href="#TestCase">Test Cases</a> [<cite><a class="bib-ref" href="#bib-test-metadata">test-metadata</a></cite>].</dd>
+                    <dt><dfn id="TestCase">Test Case</dfn></dt>
+                    <dd>..</dd>
+                  </dl>
+                </div>
+              </section>
+
+              <section id="interoperability" inlist="" rel="schema:hasPart" resource="#interoperability">
+                <h3 property="schema:name skos:prefLabel">Interoperability</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p id="interoperability-server-client">Interoperability of implementations for <a href="#Server" rel="rdfs:seeAlso">servers</a> and <a href="#Client" rel="rdfs:seeAlso">clients</a> is tested by evaluating an implementation’s ability to request and respond to HTTP messages that conform to this specification.</p>
```

Comment:
No, this section is not much more than a stub at the moment. (Server/client is copied from elsewhere and may not actually end up there - so I'll remove now to get rid of some confusion).

We first need to identify and describe the #classes-of-products. I hinted at TestReport, and have just added TechnicalReport and TestCase as other product classes - they are broadly about "content".

So then what constitutes as interoperable between products can be stated, e.g.:

TestSuite is another product class. One of the things it does is that it consumes TechnicalReports and another thing it probably does is help with the production of TestCases. It may be another specific component that consumes TRs / produces TestCases but we'll come back to that. When a TestSuite (or another well-defined product) can "consume" TRs and "produce" TestCases, that's one form of interop.

Yet another example is a processor or producer that takes TestCases as input and outputs a TestReport.

So, in essence the Solid QA document is a TR (in addition to including other information that's not pertinent to interop.)

Have a look at the updated text. Still rough. What we say is a product and expected interoperability can change.